invenio/modules/workflows/templates/workflows/details_base.html

```diff
@@ -86,141 +86,165 @@
-                    {% endfor %}
-                    <span class="label label-{{ bwobject._class}}">{{ bwobject.message }}</span>
+                <div class="modal fade" id="error_details_modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+                  <div class="modal-dialog">
+                    <div class="modal-content">
+                      <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                        <h4 class="modal-title" id="myModalLabel">{{ _('Problem with') }}: {{ bwobject.get_extra_data()['_last_task_name'] }}</h4>
+                      </div>
+                      <div class="modal-body">
+                        <p>{{ bwobject.get_error_message()[0] }}</p>
+                        <p>{{ bwobject.get_error_message()[1] }}</p>
+                      </div>
+                      <div class="modal-footer">
+                        <button type="button" class="btn btn-default" data-dismiss="modal">{{ _('Close') }}</button>
+                      </div>
+                    </div>
+                  </div>
                 </div>
               </div>
             </div>
-          </div>
+            {% endif %}
+          {% endblock %}
 
-          {% if history_objects %}
+          {% block workflow_information %}
             <div class="panel panel-info">
               <div class="panel-heading">
-                <h3 class="panel-title">{{ _('History') }}</h3>
+                <h3 class="panel-title">{{ _('Other information') }}</h3>
               </div>
               <div class="panel-body">
                 <h6 class="pull-right">
-                  <a role="button" style="cursor: pointer" data-toggle="collapse" data-target="#History">{{ _('More') }}<span class="caret"></span></a>
+                <a role="button" style="cursor: pointer" data-toggle="collapse" data-target="#record-metadata-table">
+                  {{ _('More') }}
+                  <span class="caret"></span>
+                </a>
                 </h6>
                 <div class="row"></div>
-                <div id="History" class="collapse">
+                <div id="record-metadata-table" class="collapse">
                   <div class="table table-striped">
-                    {% for object in history_objects %}
-                      {% if  object.id == bwobject.id %}
-                        <a href="#">{{ _('current') }} </a> -
-                        {% else %}
-                        <a href="{{ url_for('holdingpen.details', objectid=object.id) }}" >{{ object.id }}</a> -
-                      {% endif %}
-                      <span class="label label-{{ object._class }}">{{ object.message }}</span>
-                     {{ pretty_date(object.modified) }}
-                    <hr>
-                    {% endfor %}
+                      {% for key, value in info.items() %}
```

Comment:
why aren't you using `table` rows here if https://github.com/inveniosoftware/invenio/pull/2586/files#diff-05794fe86f3fba5b0c0d667b0b7cf7a7R201 defines classes used for table? IMHO compact table would be nicer here.